### ISSUE TITLE
Fix typos and wording in UI components

### ISSUE THREAD

**Issue by SerpentBytes** [DevelopingSpace/starchart#381]
## Description

There are instances of typos where they should not exist (for example: `aria-label`), which would make our web application less accessible. Also, I found instances where we could be more concise with our wording in specific UI components, like `tooltip`, which could improve the appearance of our web app on smaller viewports. 

We should fix this when we can.

**Comment by humphd** [comment]
What's wrong with `aria-label`?

**Comment by humphd** [comment]
Spoke with @SerpentBytes on Slack, I misunderstood that it was with `aria-label` itself, but it's with values assigned to `aria-label`, which he's going to indicate here.

**Comment by SerpentBytes** [comment]
https://github.com/DevelopingSpace/starchart/blob/main/app/components/domains-table.tsx#L150
